### ISSUE TITLE
Server offline

### ISSUE THREAD

**Issue by Drobac** [GTNewHorizons/NewHorizons#363]
Offline and not restarting

**Comment by Dream-Master** [comment]
Sorry can't do anything now not have access from the work to minecraft.
I am back home in round about 8 hours.

**Comment by Namikon** [comment]
Will check in ~30 minutes

**Comment by Namikon** [comment]
Ok, It seems that the HQM file was damaged, or HQM was updated and you forgot to mention it in the changelog.
I found this thread:
http://minecraft.curseforge.com/forums/jadeds-packs/agrarian-skies-2/questions/general/902-crashes-after-updating-to-1-0-39

Strange is, that it started fine yesterday, but when it was supposed to restart at 4:00 am, it entered the crash-loop. So something has changed, hopefully utilityworlds is not the cause, since that's the only change we've made after the server came back online yesterday. We created both void dims.

I removed the hqm save file, which will reset everyone to zero, and the Server started just fine. However there are (again) some potion IDs overwritten. We need to check the logfiles better before going live. And I suggest to only update on Fridays
